docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
```

Comment:
Collections are generally referenced by their slug instead of id (think a user accessing  https://addons.mozilla.org/en-US/firefox/collections/mozilla/webdeveloper/).  Indeed, if the specification from Product in the PRD is acted upon then the slug will be only thing stopping a 'private' collection from being known will be the known-ness of the slug, so being able to access the collection with a simple numeric ID is insecure.  But a slug isn't unique across all collections, so we need to have both collection slug and username - (i.e. every request would contain the `user` parameter... so why not just have it in the url was my thinking).

So with `user` as a parameter it would be `GET /api/v3/collections/collection/webdeveloper/?user=mozilla` for `https://addons.mozilla.org/en-US/firefox/collections/mozilla/webdeveloper/`